docs/content/concepts/partitions-schedules-sensors/partitioned-assets.mdx

```diff
@@ -0,0 +1,383 @@
+---
+title: Partitioned assets and jobs | Dagster
+description: Partitioned assets and jobs enable launching backfills, where each partition processes a subset of data.
+---
+
+# Partitioned assets and jobs
+
+<Note>
+  This page is specific to <strong>Software-defined Assets (SDAs)</strong>.
+  Looking for ops? Refer to the{" "}
+  <a href="/concepts/partitions-schedules-sensors/partitioned-op-jobs">
+    Partitioned op jobs
+  </a>{" "}
+  documentation.
+</Note>
+
+A [Software-defined Asset](/concepts/assets/software-defined-assets) (SDA) can represent a collection of _partitions_ that can be tracked and materialized independently. In many ways, each partition functions like its own mini-asset, but they all share a common materialization function and dependencies. Typically, each partition will correspond to a separate file or a slice of a table in a database.
+
+A common use is for each partition to represent all the records in a data set that fall within a particular time window, e.g. hourly, daily or monthly. Alternatively, each partition can represent a region, a customer, an experiment - any dimension along which you want to be able to materialize and monitor independently. An asset can also be partitioned along multiple dimensions, e.g. by region and by hour.
+
+A graph of assets with the same partitions implicitly forms a partitioned data pipeline, and you can launch a run that selects multiple assets and materializes the same partition in each asset.
+
+Once an asset has a set of partitions, you can launch materializations of individual partitions and view the materialization history by partition in the UI.
+
+---
+
+## Prerequisites
+
+Before continuing, you should be familiar with:
+
+- [Software-defined Assets](/concepts/assets/software-defined-assets)
+- [Jobs](/concepts/ops-jobs-graphs/jobs)
+- [I/O managers](/concepts/io-management/io-managers) (Optional; advanced)
```

Comment:
if it's optional, should it not be a prereq?

we could have a callout note in the relevant sections instead of including it in the prereqs.